language_detector/main.py

```diff
@@ -1,8 +1,28 @@
 # -*- coding: utf-8 -*-
 
 """This is the entry point of the program."""
+from .languages import LANGUAGES
+import re
 
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    # counter var created for each language, initiated with '0'
+    dictionar_of_languages = {}
```

Comment:
Over here an alternative could have been to combine these steps into a dictionary comprehension.